README.md

```diff
@@ -136,13 +136,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: baileyjm02/markdown-to-pdf@v1
+      - uses: baileyjm02/markdown-to-pdf@v1.0.1
```

Comment:
Hi @francisdb, thanks for taking the time to look at this. I will be deploying a fix shortly which will fix the versioning issue, but thanks for catching the readme variable naming.
```suggestion
      - uses: baileyjm02/markdown-to-pdf@v1
```